examples/00-fluent/exhaust_system.py

```diff
@@ -627,8 +627,8 @@
 # the images are large enough.
 
 picture = solver.tui.display.set.picture
-# use-window-container TUI option not available inside containers
-if not solver.connection_properties.inside_container:
+# use-window-container TUI option not available inside containers or ansyslab environment
```

Comment:
Consider using the proper noun in the description. I'm not sure if that is two words (Ansys Lab) or one word (AnsysLab).

Reply:
Thanks, Ansys Lab is two words as per [this page](https://fuji.ansysapis.com/docs/status/catalog/applications/ansys_lab/).